install/README.md

```diff
@@ -20,6 +20,19 @@ clusters.
 
 ## Installing Knative
 
+Knative depends on an Ingress/Gateway which is capable of routing requests to Knative Services.
+
+Currently, two options exist which provide this functionality: 
+[Istio](https://istio.io/), the Envoy-based Service Mesh, and [Gloo](https://gloo.solo.io/), the Envoy-based API Gateway created by [Solo.io](www.solo.io).
+
+[Install with Gloo](Knative-with-Gloo.md): Gloo functions as a lightweight gateway for Knative. Choose this option if you don't require service  mesh in your cluster and want a lighter-weight alternative to Istio. Note that *Knative-Eventing* 
+is not supported by Gloo at this time.
+
+[Install with Istio](#Installing-Knative-with-Istio) Istio is a popular service mesh that includes a Knative-compatiblle
```

Comment:
Create a heading, ## Installing Knative with Gloo, and put the info about Gloo there along with the link to the guide.

Move the information about Istio below, into the "Installing Knative with Istio" section. Remove the link.